src/tools/processData.js

```diff
@@ -2,11 +2,13 @@
 /* eslint-disable quotes */
 /* eslint-disable no-restricted-syntax */
 /* eslint-disable no-loop-func */
+/* eslint-disable no-await-in-loop */
 import { Op } from 'sequelize';
 import cheerio from 'cheerio';
 import faker from '@faker-js/faker';
 import {
-  ActivityReport, User, Recipient, Grant, File, Permission, RequestErrors, sequelize,
+  ActivityReport, User, Recipient, Grant, File, Permission, RequestErrors,
+  GrantNumberLink, MonitoringReviewGrantee, MonitoringClassSummary, sequelize,
```

Comment:
In other files once it complains about too many, each one should be on its own line. I would prefer it to be consistent.

Reply:
Adjusted

Reply:
Thanks for the comment. This concern is beyond the current scope, and historically, it hasn’t been an issue. We’ll monitor it going forward.